terraform/main.tf

```diff
@@ -38,7 +38,7 @@ module "aws_deploy-ap-southeast-1" {
   instance_type = "m4.large"
 
   epoch = {
-    version = "0.19.0"
+    package = "https://s3.eu-central-1.amazonaws.com/aeternity-epoch-releases/epoch-latest-ubuntu-x86_64.tar.gz"
```

Comment:
Does this terraform integration print the hash of downloaded artefact (minimum)?

Reply:
terrafrom do not download this file, it's isntance that do this on boot, can add this there

Reply:
terraform it self do not touch this file, it's used with `user_script` in boot of instance, i can add there hash if required

Reply:
Not required. (I see it is release bucket so not many re-upload.)